### PR TITLE
Introduction to the payments and the two networks 

### PR DESCRIPTION
**Pull request by johnsBeharry**
```meta
type: page
chapter: Payments
title: Networks
permalink: /guide/payments/networks
file: TBD
```

### Problem

Payments can be facilitated on both the bitcoin blockchain and lightning network.

#### How should these two networks be referred to?
1. Bitcoin is usually described as a blockchain.
2. Lighting Networks is described as a "payments network".
3. Sometimes they are referred to as onchain / offchain
4. Off-chain can also mean side-chain e.g. liquid

#### Assumptions about the reader
1. Brand new to bitcoin
2. Thinks bitcoin is "too slow" for payments
2. Thinks bitcoin is anonymous
3. Is not technical and may get easily overwhelmed

### Solution
1. When mentioning bitcoin's onchain speed refer to the security and finality of transactions as the reason.
2. Made analogies using familiar concepts like banking and mobile money
3. Lightning Network is the technology. Lightning Payments is the user benefit. Focus on the user benefit.

### Result

[Preview the introduction](https://deploy-preview-62--sad-borg-390916.netlify.app/guide/payments/introduction/) page of the payments section.

Seeking feedback and concept acknowledgments.

### CODE REVIEW

**Comment by pavlenex** [comment]
Some general feedback, I understand this is still WIP in a way, so perhaps some of the points are already considered and some points may come from my misunderstanding on what's covered in other section of the guides.

1. After `Payments` some context should be given on what this page is about.
2. Transaction goes straight into on-chain vs lightning which I think is very big leap. I suggest we first explain:
  - What is a transaction?
  - Input and outputs
  - Transaction fee (can be used as a pointer to distinguished on vs offchain later on)
  - Types of transaction
    - Onchain
      - Inputs vs output
      - Transaction verification
    - Off chain (Lightning)
3. Not really a fan of lightning analogy, but you mentioned in private you're not keen on it either.
This is very rough structure and needs quite a bit of discussion, depending what we're trying to achieve.

**Comment by yantiarifin** [comment]
Regarding the two networks, a high level explanation of why there are 2 networks may be helpful, otherwise a new user may feel they are not technical enough for bitcoin. Additionally any opportunities to use words related to money transactions which any user would understand may be helpful. For example sending and receiving instead of input and output. Also clarifying that a verification is a receipt, etc. As always, visuals would help such as flow diagrams of how transactions are processed. Would there be a brief primer on what a wallet is? It think it would be helpful too.

**Comment by GBKS** [comment]
To add on to @pavlenex comment, I'd start one level higher at the lifecycle of a transaction in the context of human interaction. Then it's easier to dive into each one of those interactions (invoice creation, transaction construction, first confirmation...). 

Here's a quick mock-up:
transaction-lifecycle-201210

File: guide/payments/introduction.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of bitcoin.
+
+#### What about privacy?
+
+Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still super important though since every time you make a payment the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
+
+We should inform our users of these risks but also provide mechanisms that help them gain an acceptable amount of privacy[^1].
+
+<<<<<<< HEAD
```
Comment (pavlenex) [inline guide/payments/introduction.md:58]:
```suggestion
```

File: guide/payments/introduction.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+We are for designing the next evolution of money. One which is internet based, and global.
+
+> **Getting Started / Why bitcoin is unique**
+>
+> Bitcoin can be used by anyone with an internet connection, which is available to around [53.6% of the world population](https://en.wikipedia.org/wiki/Global_Internet_usage), a number that is growing quickly. Some aspects of bitcoin, like key and address generation, even work without being online. This makes it the first truly globally available form of money that is unbound by country borders. People from around the world can freely transact without having to rely on middlemen like currency exchanges and local payment services.
+
+## Bitcoin is extensible, it's not just a payment network
+
+It should be noted, that bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrasutructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
```
Comment (pavlenex) [inline guide/payments/introduction.md:42]: Capital B, as I believe you're reoffering to the protocol here.
```suggestion
It should be noted, that Bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrastructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
```

File: guide/payments/introduction.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Bitcoin is secure and accessible to all
+
+Some people may have the perception that bitcoin is anonymous. This isn't quite accurate because all transactions are public so anyone can see the entire history of transfers that ever happened going back to the very first one on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
```
Comment (pavlenex) [inline guide/payments/introduction.md:46]:
```suggestion
Some people may have the perception that Bitcoin is anonymous. This isn't quite accurate because all transactions are public so anyone can see the entire history of transfers that ever happened going back to the very first one on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
```

File: guide/payments/introduction.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+#### Why are transactions public?
+
+They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of bitcoin.
```
Comment (pavlenex) [inline guide/payments/introduction.md:50]:
```suggestion
They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of Bitcoin.
```

File: guide/payments/introduction.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+#### What about privacy?
+
+Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still super important though since every time you make a payment the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
```
Comment (pavlenex) [inline guide/payments/introduction.md:54]: A suggestion but you may want to break this lengthy sentence down into a few shorter ones to make it easier to follow who's doing what.
```suggestion
Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still important though since every time you make a payment, the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
```

File: guide/payments/introduction.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+We should inform our users of these risks but also provide mechanisms that help them gain an acceptable amount of privacy[^1].
+
+<<<<<<< HEAD
+
+## Bitcoin is not too slow for payments
+
+Transactions made on-chain can take minutes or days to confirm — depending on the fee you pay. This is because they get secured by the entire network of nodes and miners which exchange transactions they receive from wallets and other services with each other. Every 10 minutes miners validate the payments within the transactions and publish a block. For this service, they typically take the transactions with the highest fees.
```
Comment (pavlenex) [inline guide/payments/introduction.md:62]: This seems a bit techy. What's on-chain? Did reader had a resource before this to intro themselves with the concept, if so then it's fine.

File: guide/payments/introduction.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone accross the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.
+
+### Lightning Payments — A micro-payment network for every day payments
+
+To meet the demands of our modern economy — movement of money needs to be borderless, secure, fast, and cheap.
+
+The Lightning Network is part of the bitcoin ecosystem that will power this economy. It allows for near-instant payments at much lower fees. This is possible because extends bitcoin in a way that does not record each payment on-chain.
+
+#### A Network of Payment Channels and Settlement 
+
```
Comment (pavlenex) [inline guide/payments/introduction.md:73]: Probably some sort of visual of LN/Channels is needed here, as at this point it gets quite hard to follow up if you're newbie imo, especially since this is intro page.

**Review by pavlenex — COMMENTED** [review]
Great progress Johns :rocket: 
Since I wasn't aware on the state of the PR, I've made some minor suggestions.

File: guide/payments/introduction.md (hunk 2 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:42]: Nice catch on the spelling. I think we settled to use lowercase bitcoin though, was part of "the first great debate" some months ago. Quite divisive lol

Comment (pavlenex) [reply guide/payments/introduction.md:30]: But even in the guide it says that when you're reffering to the protocol it's capital B https://bitcoin.design/guide/getting-started/technology-primer/#is-it-bitcoin-or-bitcoin so would be good to implement what we're preaching 😃

Comment (GBKS) [reply guide/payments/introduction.md:30]: "Getting started" is based on the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Help:Introduction):

> Since Bitcoin is both a currency and a protocol, capitalization can be confusing. Accepted practice is to use Bitcoin (singular with an upper case letter B) to label the protocol, software, and community, and bitcoins (with a lower case b) to label units of the currency.

I'd keep it simple and stick with that.

File: guide/payments/introduction.md (hunk 5 above)
Comment (GBKS) [reply guide/payments/introduction.md]: I am starting to think that we need a section that explains this more. The fact that I can see the history of your bitcoin, does not mean I can make sense of it. How do I know who is behind each address? It takes a huge effort to deanonymize addresses, so I think it might be helpful to clarify this so people are not overly paranoid.

I'd also be interested in some more data in how effective privacy-preserving techniques are. Does avoiding address re-use make me 99% safe from chain analysis? Or is mixing to an anonymity set of 50 the standard?

File: guide/payments/introduction.md (hunk 6 above)
Comment (GBKS) [reply guide/payments/introduction.md:49]: Might be also good to put this in perspective with the traditional banking system. From the [Wells Fargo FAQ](https://www.wellsfargo.com/help/online-banking/transfers-faqs/):

> Payments to your Wells Fargo credit account from a non-Wells Fargo account usually take 3-4 business days to complete.

So yes, paying with credit card at the grocery store is a bit impractical. But for many other uses, it's pretty hella fast for a global payment network.

File: guide/payments/introduction.md (hunk 7 above)
Comment (GBKS) [reply guide/payments/introduction.md:60]: Agreed. Maybe describe it as another peer-to-peer network on top of Bitcoin. It's more flexible and allows for direct payments that don't require the whole network to confirm and process them. And that's how it manages to be faster and cheaper. Or something like that.

File: guide/payments/introduction.md (hunk 2 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:30]: Aha. Ok I interpreted it as lowercase everything.

File: guide/payments/introduction.md (hunk 6 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:49]: @pavlenex do you think the intro of this page would be a ok place to introduce it?

@GBKS I kind of have something similar below:

> This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone across the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.

I learned that the local bank transfer speed / cost vary quite a bit globally. For example within Indonesia and Nigeria payments between any bank is instant, but in other places it could be 2-5 days as you mentioned. The one that always takes long is with the SWIFT network though. See next line

> This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone accross the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.

File: guide/payments/introduction.md (hunk 7 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:60]: @pavlenex yesss time for abacus beads? 😄 

@GBKS check the following paragraphs. I attempted to explain why it is faster there, and talk about two people locking up funds on-chain, then sending back and forth with one another. Then talk about final settlements on-chain.

Comment (pavlenex) [reply guide/payments/introduction.md:60]: Quite honestly abacus beads were so helpful when I tried to understand how channels works so go for it :smile:

File: guide/payments/introduction.md (hunk 6 above)
Comment (pavlenex) [reply guide/payments/introduction.md:49]: > @pavlenex do you think the intro of this page would be a ok place to introduce it?

Good question. Personally, I'd keep it separate as user dives deep into the guide. I am scared that we may overwhelm people with the concept in the introduction. , I'd explain the LN once basics of the on-chain are layed out. I kinda like the [structure Daniel suggested](https://github.com/BitcoinDesign/Guide/discussions/98#discussioncomment-284887). What is your view on this, @johnsBeharry?

File: guide/payments/introduction.md (hunk 5 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md]: Unfortunately even with carefully selecting UTXOs to spend, you may still end up using those at an exchange which is KYCed. Or someone else may doxx one of your addresses you shared with them by publishing it online. So there's so silver bullet for privacy, it's also something thats not really quantifiable.

Also, it is worth making people more paranoid in my opinion. The mere fact that we struggle with it currently and we are in the know, interacting with more advanced tools than the wider population is aware of means that we're taking autonomy from the users.

Comment (johnsBeharry) [reply guide/payments/introduction.md]: @GBKS I was also thinking some of this content can also live in the Getting Started. I'm fine with moving some more generally applicable parts over there and link over to it.

File: guide/payments/introduction.md (hunk 6 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:49]: During conversations developing/testing payment content in the mentorships calls I've had, participants would quite early unknowingly raise questions relating to lightning (speed and cost etc). This is the point where some people even look at other networks because "bitcoin is too slow" (hence why I put that heading). I tried to capture the flow of those sessions on this page. It doesn't mean I will immediately be writing up lightning content though that research is still on-going, but giving awareness of it at this point makes sense to me so the reader has a more fuller understanding of the ecosystem.

During the design process people first Googling screenshots of payment and bitcoin apps. I even spoke with a designer who designed a bitcoin wallet but never used one themselves.

Admittedly, this page has grown quite long, and the reading experience certainly needs to be improved (i hear you on more visuals for sure). Perhaps it could also be moved entirely to `guide/payments/networks` or `guide/payments/getting-started` I'm happy if we also move more generally applicable parts over to `guide/getting-started` then payments could focus more on the user flow. Thoughts? @GBKS @pavlenex

File: guide/payments/introduction.md (hunk 7 above)
Comment (johnsBeharry) [reply guide/payments/introduction.md:60]: Abacus time!

(added an illustration)

**Comment by johnsBeharry** [comment]
Closing off this issue as it's become stale. The content will be adapted in some way in the new proposed payments section structure #98
